client/src/App/App.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { Route, Switch } from 'react-router-dom';
+import './App.css';
+import Home from './pages/Home';
+import List from './pages/List';
+
+class App extends Component {
```

Comment:
Please refactor this class component to make it a functional component.

In general, try to write functional instead of class components in React - functional components are easier to read and to write tests for. As of React v.16.8, there's no benefit to writing class components anymore (you can use hooks to make functional components stateful)

Link: https://habr.com/en/post/443500/